_posts/2019-02-27-stripe-connect-part-1.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: "Implementing Stripe Connect in Rails | Part 1"
+date: 2019-02-27 16:00:00
+categories: ["rails", "stripe"]
+author: "luciano"
+---
+
+In a recent project for [OmbuLabs](https://www.ombulabs.com), we were looking for a payment solution that allows users of our platform to easily start accepting online payments but at the same time we wanted to take a fee from it.
+That's where [Stripe Connect](https://stripe.com/connect) came into place.
+
+<!--more-->
+
+To make things easier to follow we will break this topic in two articles:
+- Part 1: How to allow your users link a Stripe account so they can accept payments.
+- Part 2: How your users can receive payments in their Stripe account and how you can receive a fee for that into your own Stripe account.
+
+
+## Part 1: How to allow your users link a Stripe account so they can accept payments
+
+One of the first things you have to decide is which [Stripe account type](https://stripe.com/docs/connect/accounts) you want for your users. Each option works better for different needs. For this article we'll be using [Express accounts](https://stripe.com/docs/connect/express-accounts) (the following steps can change a bit if you decide to use a different account type)
```

Comment:
(the following steps can change a bit if you decide to use a different account type).